prov/efa/test/efa_unit_test_ep.c

```diff
@@ -915,3 +915,25 @@ void test_efa_rdm_ep_user_zcpy_rx_unhappy_due_to_sas(struct efa_resource **state
 
 	test_efa_rdm_ep_use_zcpy_rx_impl(resource, false);
 }
+
+void test_efa_rdm_ep_close_discard_posted_recv(struct efa_resource **state)
+{
+	struct efa_resource *resource = *state;
+	char buf[16];
+	int err;
+
+	efa_unit_test_resource_construct(resource, FI_EP_RDM);
+
+	/* Post recv and then close ep */
+	fi_recv(resource->ep, (void *) buf, 16, NULL, FI_ADDR_UNSPEC, NULL);
```

Comment:
I don't know if we scan the efa tests for coverity but if we do I think this might get picked up - it doesn't like ignoring the return codes (here and on fi_close before). Technically should check this and only continue on FI_SUCCESS

Reply:
Thanks, updated